src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,12 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Ball[] balls = new Ball[3];
+        for (int i = 0; i < 3; i++) {
+            balls[i] = new Lottery().getRandomBall();
```

Comment:
let's create object of `new Lottery()` before the loop and use it in the loop
```suggestion
            balls[i] = lottery.getRandomBall();
```

Reply:
preferable way) 
because when we use `new Lottery()` in for loop we create a new object during each iteration. So when the number of iterations is large, we create a lot of extra objects. We should avoid it